pkg/morph/client/nns.go

```diff
@@ -101,26 +101,26 @@ func (c *Client) InitSidechainScope() error {
 func autoSidechainScope(ws *rpcclient.WSClient, conf *cfg) error {
 	nnsHash, err := nns.InferHash(ws)
 	if err != nil {
-		return fmt.Errorf("InferHash: %w", err)
+		return fmt.Errorf("resolve NNS contract address: %w", err)
 	}
 
 	var nnsReader = nns.NewReader(invoker.New(ws, nil), nnsHash)
 
 	balanceHash, err := nnsReader.ResolveFSContract(nns.NameBalance)
 	if err != nil {
-		return fmt.Errorf("resolving balance: %w", err)
+		return fmt.Errorf("resolve Balance contract address by NNS domain name %q: %w", nns.NameBalance, err)
 	}
 	cntHash, err := nnsReader.ResolveFSContract(nns.NameContainer)
 	if err != nil {
-		return fmt.Errorf("resolving container: %w", err)
+		return fmt.Errorf("resolve Container contract address by NNS domain name %q: %w", nns.NameContainer, err)
```

Comment:
"NNS domain name" confuses me. seems like it is about some ".nns"

Reply:
not sure i got ur point, pls write message text u suggest